manim/scene/scene.py

```diff
@@ -78,20 +80,16 @@ def __init__(self, renderer=None, **kwargs):
             random.seed(self.random_seed)
             np.random.seed(self.random_seed)
 
-        self.setup()
-
     def render(self):
         """
         Render this Scene.
         """
-        self.original_skipping_status = config["skip_animations"]
+        self.setup()
         try:
             self.construct()
         except EndSceneEarlyException:
             pass
         self.tear_down()
-        # We have to reset these settings in case of multiple renders.
-        config["skip_animations"] = self.original_skipping_status
```

Comment:
This part of the diff is messy so let me explain. There are two things going on:

1. I moved `setup()` from `__init__` to `render()`. Seemed more appropriate IMHO (lines 81 and 87).
2. `config` no longer has `skip_animations` so it does not need to be restored (lines 87 and 93-94).